modules/core/src/main/java/org/apache/ignite/internal/ClusterMetricsSnapshot.java

```diff
@@ -823,12 +823,12 @@ public void setCurrentIdleTime(long curIdleTime) {
 
     /** {@inheritDoc} */
     @Override public float getBusyTimePercentage() {
-        return 1 - getIdleTimePercentage();
+        return Math.max(100 - getIdleTimePercentage(), 0);
```

Comment:
Let's replace 100 with 1 here to leave public contract unchanged.

Reply:
Again, the method is called "Percentage"